2.10/2.10.18/index.html

```diff
@@ -23,26 +23,25 @@
 	alert('Поставьте будильник на утро');
 
 	hours = prompt('Укажите часы');
-
 	minute = prompt('Укажите минуты');
 
 	if (hours >= 0 && hours <= 24 && hours !== '') {
 		if (hours < 10) {
-			hours = `0`+ `${hours}`;
+			hours = '0' + hours;
 		}
 	} else {
 		hours = '00';
 	}
 
 	if (minute >= 0 && minute <= 59 && minute !== '') {
 		if (minute < 10) {
-			minute = `0`+ `${minute}`;
+			minute = 0 + minute;
```

Comment:
`'0' + minute` will be more understandable. Any developer will understand quickly that is a concatenation, not math operation